tests/test_fixtures.py

```diff
@@ -1,30 +0,0 @@
-from __future__ import absolute_import, print_function, unicode_literals
```

Comment:
What is the reason behind deleting these tests?

Reply:
When python 3 was first released, python introduced future functions into python 2.x to help projects transition to the new version but still support python 2, we don't need the support anymore